packages/dashboard-frontend/src/pages/GetStarted/updateDevfileMetadata.ts

```diff
@@ -19,7 +19,7 @@ import {
 import getRandomString from '../../services/helpers/random';
 
 export function updateDevfileMetadata(devfile: api.che.workspace.devfile.Devfile, meta?: che.DevfileMetaData): api.che.workspace.devfile.Devfile {
-  if (isDevfileV2(devfile)) {
+  if (isDevfile(devfile)) {
```

Comment:
well, it seems strange... You check if devfile is devfile, for sure it is )
please consider making things clearer here

Reply:
> You check if devfile is devfile, for sure it is )

Not necessarily. 
The object named `devfile` may have a shape different from `Devfile`. This typeguard checks if the object has all required fields so it can be then used safely as `Devfile`.

I was also hesitating about this naming but decided that if this function is explicitly imported from the `devfileApi` module then it's clear enough what it is supposed to do. My assumption was wrong :)

So, let me rename this typeguard to `isDevfileV2`. And should I rename the interface in the same way `devfileApi.Devfile` to `devfileApi.DevfileV2`?